src/components/VerticalNavigation/__mocks__/mockIconBarChildren.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export const mockIconBarChildren = (
+  <React.Fragment>
+    <li className="drawer-pf-trigger">
+      <a className="nav-item-iconic">
+        <span className="fa fa-bell" title="Notifications" />
+        <span className="badge badge-pf-bordered" />
+      </a>
+    </li>
+    <li className="dropdown">
+      <a className="dropdown-toggle nav-item-iconic" id="dropdownMenu1">
+        <span title="Help" className="fa pficon-help" />
+        <span className="caret" />
+      </a>
+      <ul className="dropdown-menu">
+        <li>
+          <a href="#0">Help</a>
+        </li>
+        <li>
+          <a href="#0">About</a>
+        </li>
+      </ul>
+    </li>
+    <li className="dropdown">
+      <a
+        href="#0"
```

Comment:
This `href=#0` wreaks havoc and sends me off to another component, messes up the storybook navigation.

Reply:
I don't see that behavior, but I didn't build this locally.